# Generated by Django 5.1.1 on 2024-10-08 13:01

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('api', '0001_initial'),
    ]

    operations = [
        migrations.AddField(
            model_name='theuser',
            name='role',
            field=models.CharField(choices=[('E', 'Employer'), ('J', 'Jobseeker')], default='J', max_length=1),
            preserve_default=False,
        ),
        migrations.AlterField(
            model_name='theuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 20 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=20, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
